File: programacao_orientada_a_objetos/Aula_1/AulaUm.java
package UFOPA.programacao_orientada_a_objetos.Aula_1;

import java.util.Scanner;

public class AulaUm {
    public static void main(String[] args) {
        Scanner variavel = new Scanner(System.in);
        System.out.println("________________________");
        System.out.println("Triângulo X");
        System.out.println("________________________");
        System.out.print("1° Lado: ");
        double xLadoUm = variavel.nextDouble();
        System.out.print("2° Lado: ");
        double xLadoDois = variavel.nextDouble();
        System.out.print("3° Lado: ");
        double xLadoTres = variavel.nextDouble();
        double semiperimetroX = (xLadoDois + xLadoUm + xLadoTres) / 2;
        double xArea = Math.sqrt(semiperimetroX * (semiperimetroX - xLadoUm) * (semiperimetroX - xLadoDois) * (semiperimetroX - xLadoTres));
        System.out.printf("Área de X: %.4f", xArea);

        System.out.println("\n________________________");
        System.out.println("Triângulo Y");
        System.out.println("________________________");
        System.out.print("1° Lado: ");
        double yLadoUm = variavel.nextDouble();
        System.out.print("2° Lado: ");
        double yLadoDois = variavel.nextDouble();
        System.out.print("3° Lado: ");
        double yLadoTres = variavel.nextDouble();
        double semiperimetroY = (yLadoDois + yLadoUm + yLadoTres) / 2;
        double yArea = Math.sqrt(semiperimetroY * (semiperimetroY - yLadoUm) * (semiperimetroY - yLadoDois) * (semiperimetroY - yLadoTres));
        System.out.printf("Área de X: %.4f", yArea);
        
        System.out.println("\n________________________");

        if (xArea == yArea) {
            System.out.print("Ambos triângulos são iguais.");
            System.out.println("________________________");
        } else if (xArea > yArea) {
            System.out.println("O maior triângulo é X");
            System.out.println("________________________");
        } else {
            System.out.println("O maior triângulo é y");
            System.out.println("________________________");
        }
        variavel.close();
    
    }

}

File: programacao_orientada_a_objetos/Aula_1/aula_um.py
print(30 * '-')
print('Triângulo')
print(30 * '-')

lado_um = float(input('1° Lado: '))
lado_dois = float(input('2° Lado: '))
lado_tres = float(input('3° Lado: '))
semiperimetro = (lado_dois + lado_tres + lado_um) / 2
areaX = (semiperimetro * (semiperimetro - lado_um) * (semiperimetro -  lado_dois) * (semiperimetro - lado_tres)) ** (1 / 2)
print(f'Área x: {areaX:.4f}')

print(30 * '-')

lado_um = float(input('1° Lado: '))
lado_dois = float(input('2° Lado: '))
lado_tres = float(input('3° Lado: '))
semiperimetro = (lado_dois + lado_tres + lado_um) / 2
areaY = (semiperimetro * (semiperimetro - lado_um) * (semiperimetro -  lado_dois) * (semiperimetro - lado_tres)) ** (1 / 2)
print(f'Área Y: {areaY:.4f}')

print(30 * '-')

if areaX == areaY:
    print('Os triângulos são iguais.')
elif areaX > areaY:
    print('X > Y')
else:
    print('Y > X')
print(30 * '-')